lib/SPIRV/SPIRVInternal.h

```diff
@@ -194,10 +200,19 @@ typedef SPIRVMap<SPIRAddressSpace, SPIRVStorageClassKind>
 template <>
 inline void SPIRVMap<SPIRAddressSpace, SPIRVStorageClassKind>::init() {
   add(SPIRAS_Private, StorageClassFunction);
+  add(SPIRAS_Private, StorageClassPrivate);
```

Comment:
This is not clear to me. What SPIRAS_Private supposed to be mapped to StorageClassFunction or StorageClassPrivate ? Technically the later add() call overwrites former ones.
I don't think OpenCL private address space can be mapped to StorageClassPrivate in SPIR-V, they are not the same. Maybe we could add some other kind of private in the SPIRAddressSpace enum above, like "SPIRAS_PrivateGlobal" or "SPIRAS_PrivateShader", or whatever you think is suitable.